src/bin/cargo/commands/owner.rs

```diff
@@ -7,24 +7,71 @@ pub fn cli() -> Command {
     subcommand("owner")
         .about("Manage the owners of a crate on the registry")
         .arg_quiet()
-        .arg(Arg::new("crate").action(ArgAction::Set))
+        .arg(Arg::new("crate"))
+        .arg_required_else_help(true)
```

Comment:
I don't think any results should be shown without explicit user action.

Reply:
A lot of CLIs default a command to list.  However, since this is a network operation, I can see foregoing that.  As this is an error, we can change it in the future.